app/static/js/main.js

```diff
@@ -17,7 +17,7 @@ $.ajaxSetup({
 $(document).ready(function () {
   // Keep track of mailto: links as connections
   $("a[href^=mailto]").click(function (evt) {
-    var connectTo = $(evt.target).data('connect-to');
+    var connectTo = $(this).data('connect-to');
```

Comment:
The "match me" pages wrap an `<i>` inside a `mailto:` link, which means that the target of click events is the `<i>` rather than the link. 

The existing JS assumes that the event target is always the link, however, which is why our code wasn't working.